src/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ option(CMAKE_POSITION_INDEPENDENT_CODE "Position independent code" ON)
 
 include(cmake/compiler_flags.cmake)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
nitpick: [Kitware](https://gitlab.kitware.com/cmake/community/wikis/doc/tutorials/How-To-Find-Libraries#writing-find-mod) suggests `cmake/Modules`

Reply:
I'm quite sure you don't mind, I'll quickly change it. Feel free to revert if you really don't like it :laughing: